<ui version="4.0" >
 <class>EditDialog</class>
 <widget class="QWidget" name="EditDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit new filename</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>431</width>
     <height>75</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" >
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="leftLabel" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Filename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>-></string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="rightEdit" />
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QPushButton" name="fromFilenameButton" >
        <property name="text" >
         <string>From &amp;Filename</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fromTagsButton" >
        <property name="text" >
         <string>From &amp;Tags</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="okButton" >
        <property name="text" >
         <string>&amp;OK</string>
        </property>
        <property name="autoDefault" >
         <bool>true</bool>
        </property>
        <property name="default" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton" >
        <property name="text" >
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
